### DISCUSSION TITLE
How to change site Metadata? It's always show lupin at header

### DISCUSSION THREAD

**Discussion by muhammadizzuddin** [LekoArts/gatsby-themes#608] [Questions] [answered]
### File contents (if changed)

`gatsby-config.js`: N/A 
`package.json`: N/A 
`gatsby-node.js`: N/A 
`gatsby-browser.js`: N/A 
`gatsby-ssr.js`: N/A

**Comment by LekoArts** [comment, marked as answer]
You can find the information here: https://github.com/LekoArts/gatsby-themes/tree/master/themes/gatsby-theme-minimal-blog#additional-configuration
